alf/trainers/policy_trainer.py

```diff
@@ -178,13 +186,32 @@ def train(self):
 
                 logging.info(s.getvalue())
             self._save_checkpoint()
-        except:
-            ans = input("Do you want to save checkpoint? (y/n): ")
-            if ans.lower().startswith('y'):
-                self._save_checkpoint()
+            checkpoint_saved = True
         finally:
+            if not checkpoint_saved:
```

Comment:
The reason of not using except is that we can debug if anything go wrong.

Reply:
Will finally print out the exception message?

Reply:
Yes.